File: PHPLASProc/GeoProc/Utility.php
<?php
class Utility
{
	// public Static method
	public static function ContainsList($listList, $listItem) // listList: [[]] list, listItem: []
	{

	   if($listList == null || $listItem == null)
	   {
		  return false;
	   }

	   sort($listItem);

	   for ($i = 0; $i < count($listList); $i ++ )
	   {
   
		  $temp = $listList[$i]; 		  		  
		  
		  if (count($temp) == count($listItem))
		  {			  
			  
			 sort($temp);

			 $itemEqual = true;
			 for($j = 0; $j < count($listItem); $j ++ )
			 {
				if($temp[$j] != $listItem[$j])
				{
				   $itemEqual = false;
				   break;
				}
			 }

			 if($itemEqual)
			 {
				return true;
			 }
		  }
		  else
		  {			  
			 return false;
		  }
	   }

	   return false;
	}

}
?>

File: CSLASProc/GeoProc/Utility.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GeoProc
{
    class Utility
    {
        public static bool ContainsList<T>(List<List<T>> list, List<T> listItem)
        {
            listItem.Sort();

            for (int i = 0; i < list.Count; i++)
            {

                List<T> temp = list[i];

                if (temp.Count == listItem.Count)
                {
                    temp.Sort();

                    if (temp.SequenceEqual(listItem))
                    {
                        return true;
                    }

                }
            }

            return false;
        }
    }
}
